app/src/lib/text-token-parser.ts

```diff
@@ -280,6 +280,12 @@ export class Tokenizer {
           )
           break
 
+        case '\n':
+          this.append(element)
+          this.append(' ')
```

Comment:
@tsvetilian-ty can you explain a bit how these changes fix the issue? At first sight, I don't see how adding this space fixes the link parsing 🤔

Reply:
Oh I just saw why (I think):
https://github.com/desktop/desktop/blob/2a4d2ef3ea73af86fc5a3ad2732f4a08899d2cc9/app/src/lib/text-token-parser.ts#L215

and https://github.com/desktop/desktop/blob/2a4d2ef3ea73af86fc5a3ad2732f4a08899d2cc9/app/src/lib/text-token-parser.ts#L183

I think it would be more appropriate to just check for all whitespace characters instead of only space, using for example `/\s/.test(lastItem)` instead of `lastItem !== ' '`, or even `lastItem.trim() === ''`